docs/one-click-integrations/github-integration.md

```diff
@@ -30,10 +34,17 @@ To set up the GitHub integration:
 
     ![Pulse GitHub App installed successfully](images/github-installed-ok.png)
 
-1.  Choose the strategy to detect deployments that best fits your workflows. See the [section below](#deployment-detection-strategy) for a detailed description of each option.
+1.  Choose the strategy to detect **deployments** that best fits your workflows. See the [section below](#deployment-detection-strategy) for a detailed description of each option.
 
     ![Choosing a deployment detection strategy](images/github-strategy.png)
 
+1.  Choose the strategy to detect **incidents** that best fits your workflows. See the [section below](#incident-detection-strategy) for a detailed description of each option.
+
+    !!! note
```

Comment:
@mrfyda I also added a note here in the setup flow, although this could be seen as a UI failure.